### PR TITLE
microk8s.enable fixes

### PR DESCRIPTION
**Pull request by RefinedSoftwareLLC**
istio no longer needs a custom installer

### CODE REVIEW

**Comment by YourTechBud** [comment]
@RefinedSoftwareLLC do sign the cla so I can proceed to merge this PR

File: content/install/kubernetes/microk8s.md [hunk 1]
```diff
@@ -47,13 +37,19 @@ Wait for all the pods to start:
...
-## Step 4: Open Mission Control
+Or if you are going to access microk8s remotely from another machine, use the following command instead (replacing YOUR_IP with its value):
+
+```bash
+microk8s.kubectl port-forward -n space-cloud gateway-0 --address localhost,YOUR_IP 4122:4122
+```
+
+## Step 3: Open Mission Control
 
 You should be able to access Mission Control on `http://localhost:4122/mission-control`
```
Comment (YourTechBud) [inline content/install/kubernetes/microk8s.md:54]: Maybe we should mention that thet can use their IP as well here.

```suggestion
You should be able to access Mission Control on `http://localhost:4122/mission-control` or `http://YOUR_IP:4122/mission-control`
```

File: content/install/kubernetes/microk8s.md [hunk 2]
```diff
@@ -47,13 +37,19 @@ Wait for all the pods to start:
...
 
-The last step would be to port forward Space Cloud's ports.
+The last step would be to port forward Space Cloud's ports on localhost:
 
 ```bash
 microk8s.kubectl port-forward -n space-cloud gateway-0 4122:4122
 ```
 
-## Step 4: Open Mission Control
+Or if you are going to access microk8s remotely from another machine, use the following command instead (replacing YOUR_IP with its value):
```
Comment (YourTechBud) [inline content/install/kubernetes/microk8s.md:46]: Can you do the same for `minikube` as well?

Comment (RefinedSoftwareLLC) [reply content/install/kubernetes/microk8s.md]: We decided port 4122 access was redundant with port 30122 access.

**Pull request merged**